tabbycat/results/views.py

```diff
@@ -269,9 +269,13 @@ def get_all_ballotsubs(self):
 
     def get_form_class(self):
         if self.tournament.pref('teams_in_debate') == 'bp' and self.debate.round.is_break_round:
+            if self.debate.round.next is None:
+                return SingleBallotEliminationResultForm
             return BPEliminationResultForm
         elif self.debate.round.ballots_per_debate == 'per-adj':
             return PerAdjudicatorBallotSetForm
+        elif self.debate.round.is_break_round and self.tournament.pref('elim_ballot_nom'):
```

Comment:
I think there's an implicit `self.tournament.pref('teams_in_debate') == 'two'` in here (because the BP case is covered in the first `if` block) but it's better to be explicit if that's the intention.